router/diag_cntl.c

```diff
@@ -556,6 +598,118 @@ void diag_cntl_set_buffering_mode(struct peripheral *perif, int mode)
 	}
 }
 
+static int diag_query_pd_name(char *process_name, char *search_str)
+{
+	if (!process_name)
+		return -EINVAL;
+
+	if (strstr(process_name, search_str))
+		return 1;
+
+	return 0;
+}
+
+int diag_add_diag_id_to_list(uint8_t diag_id, char *process_name, struct peripheral *perif)
+{
+	struct diag_id_tbl_t *new_diagid = NULL;
```

Comment:
try to stay consistent, new_diag_id vs new_diagid since 'diag_id" is used

Reply:
ok